command/show_test.go

```diff
@@ -150,6 +154,92 @@ func TestShow_state(t *testing.T) {
 	}
 }
 
+func TestPlan_json_output(t *testing.T) {
+	fixtureDir := "test-fixtures/show-json"
+	testDirs, err := ioutil.ReadDir(fixtureDir)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	for _, entry := range testDirs {
+		if !entry.IsDir() {
+			continue
+		}
+
+		t.Run(entry.Name(), func(t *testing.T) {
+			inputDir := filepath.Join(fixtureDir, entry.Name())
+
+			cwd, err := os.Getwd()
+			if err != nil {
+				t.Fatalf("err: %s", err)
+			}
+			if err := os.Chdir(inputDir); err != nil {
+				t.Fatalf("err: %s", err)
+			}
+			defer os.Chdir(cwd)
+
+			p := showFixtureProvider()
+			ui := new(cli.MockUi)
+			pc := &PlanCommand{
```

Comment:
I think this may also need to run either `GetCommand` (better if possible) or `InitCommand` (if not) here in order to support tests that have child modules, so that we can init the modules manifest, but fine to wait on that until there _is_ a test that includes a child module since it shouldn't disrupt the flow here very much.